src/current/cockroachcloud/authorization.md

```diff
@@ -22,15 +22,11 @@ In CockroachDB {{ site.data.products.cloud }}, an organization corresponds to an
 CockroachDB {{ site.data.products.cloud }} has a hierarchical authorization model, where roles can be assigned at different scopes:
 
 1. Organization: Each CockroachDB {{ site.data.products.cloud }} organization has a set of roles defined on it, which allow users to perform administrative tasks relating to the management of clusters, organization users, SQL users, and billing.
-1. Folder: If an organization is enrolled in [CockroachDB {{ site.data.products.cloud }} Folders (Limited Access)]({% link cockroachcloud/folders.md %}), roles can be assigned on folders. Role inheritance is transitive; a role granted on the organization or a folder is inherited by descendent resources.
+1. Folder: Roles can be assigned on folders. Role inheritance is transitive; a role granted on the organization or a folder is inherited by descendent resources.
```

Comment:
Suggest adding a link to someplace I can read more about these 'Roles' - are they the same roles as described [here](https://www.cockroachlabs.com/docs/v23.2/security-reference/authorization#roles) or something different?  (I really hope they're not something different that is also called "roles" but this is computers after all so ...  :-} )